frontend/src/metabase/visualizations/visualizations/LinkViz/LinkViz.styled.tsx

```diff
@@ -14,7 +14,7 @@ export const DisplayLinkCardWrapper = styled.div`
 `;
 
 export const EditLinkCardWrapper = styled.div`
-  padding: 0.5rem 1rem;
+  padding: 0 1rem;
```

Comment:
the editing state looks a little silly now. Get a quick design look at this, but maybe remove the horizontal padding too?

<img width="308" alt="Screen Shot 2023-06-20 at 7 57 40 AM" src="https://github.com/metabase/metabase/assets/30528226/817e031f-280e-44c1-880c-37c44edb04e6">

Reply:
@kdoh maybe you can chime in here?